cli/cli/App.cs

```diff
@@ -671,6 +670,64 @@ protected virtual Parser GetProgram()
 		});
 		return commandLineBuilder.Build();
 	}
+
+	private async Task ProxyCommand(InvocationContext context, IDependencyProviderScope provider)
+	{
+		
+		// get the version of dotnet available (which may not always be the global dotnet installation) 
+		var dotnetPath = context.ParseResult.GetValueForOption(provider.GetService<DotnetPathOption>());
+		var isPretty = context.ParseResult.GetValueForOption(provider.GetService<ShowPrettyOutput>());
+		var appContext = provider.GetService<IAppContext>();
+		var shouldRedirect = (appContext.UsePipeOutput || appContext.ShowRawOutput);
+			
+		var argumentsToForward= string.Join(" ", new []{"beam", "--pretty"}.Concat(Environment.GetCommandLineArgs()[1..]));
+
+		var warningMessage =
+			$"You tried used a Beamable CLI version different than the one configured in this project. We are forwarding the command ({argumentsToForward}) to the version the project is using. Instead of relying on this forwarding, please 'dotnet beam' from inside the project directory.";

```

Comment:
I think there is a typo here in the `You tried used a`, right?

Reply:
yep- fixed.